lib/sidekiq/form526_backup_submission_process/submit.rb

```diff
@@ -69,9 +69,12 @@ class Submit
         raise e
       end
 
+      # rubocop:disable Metrics/MethodLength
       def perform(form526_submission_id)
         return unless Settings.form526_backup.enabled
 
+        submission = Form526Submission.find(form526_submission_id)
+        submission.update(submit_endpoint: :benefits_intake_api)
         job_status = Form526JobStatus.find_or_initialize_by(job_id: jid)
         job_status.assign_attributes(form526_submission_id:,
                                      job_class: 'BackupSubmission',
```

Comment:
Nit: this job class name I'd store in a constant

Reply:
This is beyond the scope of my updates.

Reply:
LOL *facepalm* sorry you're right that's not even you code